.github/workflows/agw-build-publish-container.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: AGW Build & Publish Container
+
+on:
+  push:
+    branches:
+      - master
+      - 'v1.*'
+  pull_request_target:
+    types: [ opened, reopened, synchronize ]
+    branches:
+      - master
+      - 'v1.*'
+
+jobs:
+
+  collect-is-successful:
```

Comment:
Perhaps a more explanatory name for this job would be helpful, how about `agw-container-build-success`?

Reply:
I would not now, why I should duplicate the information contained in the workflow name.

Reply:
"agw", "container", "build" are duplicated.
"is-successful" is collected here.
Changed.